x-pack/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/agg_select/agg_select.tsx

```diff
@@ -97,8 +94,11 @@ export const AggSelect: FC<Props> = ({ fields, changeHandler, selectedOptions, r
       isInvalid={validation.valid === false}
       data-test-subj="mlJobWizardAggSelection"
     >
-      <EuiComboBox
-        singleSelection={{ asPlainText: true }}
+      <EuiComboBoxWithFieldStats
```

Comment:
Is it possible to have an empty placeholder here? I don't think 'Any' is what we want to show inside the wizard

<img width="265" alt="Screenshot 2024-06-25 at 10 16 30" src="https://github.com/elastic/kibana/assets/7405507/1c809a29-7ca6-48ff-8daf-9fc0e5c650bb">

Or if a placeholder is needed, then 'Select an aggregation' as you have for the `aria-label` would be better.

Reply:
And maybe it's just because I'm used to the current control, but is it possible to increase the width of the empty control to closer match the current UX?

<img width="448" alt="Screenshot 2024-06-25 at 10 24 36" src="https://github.com/elastic/kibana/assets/7405507/79b3d3fa-598a-48df-9faa-94c606a1e0a4">

Reply:
Updated in latest changes